tests/main.rs

```diff
@@ -54,5 +57,15 @@ fn tests() -> TestTree {
 }
 
 fn main() {
-    default_main(Config::default(), tests())
+    let completed_tasks = default_main(Config::default(), tests());
+    let failed_tasks: Vec<CompletedTask> = completed_tasks
+        .into_iter()
+        .filter(|task| !task.status.is_success_or_skip())
+        .collect();
+
+    // In this particular test-suite, we expect two infinite loops to be stopped by raclette,
+    // in your case, you should probably ensure failed_tasks.len() == 0
+    if failed_tasks.len() != 2 {
+        std::process::exit(1);
+    }
```

Comment:
I'm not sure I understand this part of the change. `tests/main.rs` is more like an example, not an actual test. I think we should call exit(1) if there is at least one task that is not ok.

Reply:
But that main gets run when we do `cargo test`. If we call `exit(1)` the `../raclette$ cargo test` run is a failure